src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/notifier.html.twig

```diff
@@ -114,24 +114,24 @@
                                     <span class="label">Subject</span>
                                     <h2 class="m-t-10">{{ message.getSubject() ?? '(empty)' }}</h2>
                                 </div>
-                                {% if message.getNotification is defined %}
+                                {% set notification = message.notification ?? null %}
```

Comment:
That cannot work when the `getNotificaiton()` does not exist.
Should be:

```
{% set notification = message.notification is defined ? message.notification %}
```

Reply:
Is this related to specific Twig versions? I tested `??` using v3 and it worked the same way when the method was missing or when it returned `null`.

Reply:
Forget about it :)